File: Front End/src/service/doctor.ts
export interface Doctor {
    id: number;
    isToggleOn: boolean;
    firstName: string;
    lastName: string;
    degree: string;
    specialization: string;
    email: string;
    phonenumber: number;
    profilePicture: string;
    approved?: boolean;
  declined?: boolean;
  }
  

File: Back End/BigBang Assessment 2/Models/Doctor.cs
using System.ComponentModel.DataAnnotations;

namespace BigBang_Assessment_2.Models
{
    public class Doctor
    {
        [Key]
        public int Id { get; set; }

        public string? FirstName { get; set; }
        public string? LastName { get; set; }
        public string? Degree { get; set; }
        public string? Specialization { get; set; }
        public string? Email { get; set; }
        public string? PhoneNumber { get; set; }
        public string? ProfilePicture { get; set; }

        public ICollection<Patient>? Patients { get; set; }
        public ICollection<Appointment>? Appointments { get; set; }
    }
}
